docs/modules/ROOT/pages/user-code-deployment.adoc

```diff
@@ -0,0 +1,202 @@
+= User Code Deployment
+:description: pass:q[Hazelcast allows users to deploy code for their custom and domain classes. In the operator you can deploy your code from external buckets and Kubernetes ConfigMap objects. ]
+
+{description}
+
```

Comment:
I think it is a good idea to add a paragraph to sum up what the page is for. Should we do it in the description or create a new section?

About other methods, there is a way to share classes through members but we do not allow it since the members are operator by us. They can also deploy classes from the clients which we allow with the `clientEnabled` field.

Reply:
Hi there Muhammet, I think just a paragraph under the description would be fine.

Reply:
Is it OK to add this in otherwise that page is still a bit difficult to understand.

This topic explains how to configure code deployments. It also brings together examples of when you might use code deployments as part of the configuration of other features, such as the xref:hazelcast:data-structures:working-with-external-data.adoc[implementation of an external datastore] or an xref:hazelcast:computing:executor-service.adoc[executor service] for the asynchronous execution of tasks.